src/cache/file.go

```diff
@@ -68,7 +68,22 @@ func (fc *File) Get(key string) (string, bool) {
 		return "", false
 	}
 
-	if co, ok := val.(*Entry); ok {
+	if co, ok := val.(*Entry); ok && !co.Expired() {
```

Comment:
@JanDeDobbeleer is this change okay? It matches the description both here and in the `Context` interface i.e., that the value isn't returned if expired.

Reply:
@heaths they are removed on initialization so you don't have to worry about that when calling the cache.